setup.cfg

```diff
@@ -61,3 +61,4 @@ exclude_lines =
     pragma ${PLATFORM_SYSTEM}: no cover
     pragma ${OS_NAME}: no cover
     pragma Python [0-9.,]*${PYTHON_VERSION}[0-9.,]*: no cover
+    raise RuntimeError.*Unreachable code
```

Comment:
They are not all covered.  They must have a comment which says "Unreachable code" otherwise they wont be included by this rule.
"pragma: no cover" says nothing.